shared/app/index.native.tsx

```diff
@@ -92,7 +92,7 @@ const Keybase = () => {
       <Provider store={_store.store}>
         <PortalProvider>
           <SafeAreaProvider>
-            <Styles.StyleContext.Provider value={{canFixOverdraw: true}}>
+            <Styles.StyleContext.Provider value={Styles.yesFixOverdraw}>
```

Comment:
if this value mutates on render it can affect all children who consume this so instead make the object once so we don't thrash